src/java/org/apache/cassandra/io/sstable/format/big/BigFormat.java

```diff
@@ -66,7 +67,99 @@
 import static org.apache.cassandra.io.sstable.format.SSTableFormat.Components.DATA;
 
 /**
- * Legacy bigtable format
+ * Legacy bigtable format. Components and approx lifecycle:
+ * <br>
+ * {@link SSTableFormat.Components}
+ * <br>
+ * {@link Components#ALL_COMPONENTS}
+ *  <ul>
+ *     <li>
+ *       {@link Components#SUMMARY}: When searching for a PK we go here for a first approximation on where to look in the index file. It is
+ *       a small sampling of the Index entries intended for a first fast search in-memory.
```

Comment:
`for the first` probably? I am not a native speaker though.

Reply:
I wanted to say 'a' not 'the' :shrug: